deployment/gunicorn/commcarehq_wsgi.py

```diff
@@ -15,5 +15,8 @@
 patch_all(subprocess=True)
 patch_psycopg()
 
+from manage import run_patches
+run_patches()
```

Comment:
Gunicorn was failing to start up until we have patched it in `commcarehq_wsgi.py`.
This caused staging to be inaccessible for some time.

Reply:
Can the `run_patches()` in [gunicorn_conf.py](https://github.com/dimagi/commcare-hq/blob/b2fadfc628cca9a27e2791adf0d2666a26ff764b/deployment/gunicorn/gunicorn_conf.py#L17-L18) be removed since it was added here? I'm not sure if we've tested applying patches twice in the same process.